src/components/notebook-utils.js

```diff
@@ -48,8 +49,16 @@ export const notebookNameValidator = existing => ({
     message: 'already exists'
   }
 })
+// removes leading dirs and a file ext suffix on paths
+export const getDisplayName = name => stripExtension(getFileName(name))
```

Comment:
There's a lot of inconsistent usage of files (curse Javascript lack of typing...). Some places we pass around the file in the form `notebooks/fileName.ext`, some places `fileName.ext`, and some places `fileName`. This PR keeps all params in the exact same format, and adds some utilities to make it a bit clearer what exactly is being used where, but doesn't address any consistency to minimize changes/complexity overhead since these strings are used in conjunction with bucket/welder calls and displayed in various ways. We may want to clean this up a bit at some point.